docusaurus.config.js

```diff
@@ -449,9 +449,14 @@ const config = {
               existingPath.replace("/build/libraries", "/tooling"),
```

Comment:
This should be removed right? `/build/libraries` does not exists

Reply:
Ah this is the wrong way around, we need to have libraries to tooling though

Reply:
But that's libraries? Where is that?